.github/workflows/ci.yaml

```diff
@@ -210,11 +218,12 @@ jobs:
   test_java8:
     runs-on: [self-hosted, Linux]
     container: lampepfl/dotty:2020-04-24
-    if: (
+    if: "(
```

Comment:
Why this change?

Reply:
Quotes are needed when using ! apparently